pom.xml

```diff
@@ -33,7 +33,7 @@
         </license>
     </licenses>
     <properties>
-        <ddf.version>2.9.0</ddf.version>
+        <ddf.version>2.10.0-SNAPSHOT</ddf.version>
```

Comment:
There are some new webconsole changes in 2.10.0-SNAPSHOT that will need to be addressed before Alliance can move up to this version.  We can either address them here, or bump the version in a separate ticket.  See : https://github.com/codice/ddf/commit/25955014b299aa57ea6c69967fdbcc9aaf9db462